packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/SQLEditor.tsx

```diff
@@ -31,6 +31,8 @@ export function SQLEditor({ table }: Props) {
       query,
     },
   });
+  const currentQuery = form.watch("query");
+  const hasUnsavedChanges = currentQuery !== query;
```

Comment:
similar to above, maybe rename?
```suggestion
  const hasQueryChanged = currentQuery !== query;
```
or just inline this ternary below since we don't use it anywhere else

Reply:
Yeah agreed, inlining is better in this case, adjusted